Providers/Scripts/2.4x-2.5x/Scripts/nxAvailableUpdates.py

```diff
@@ -134,7 +134,7 @@ def GetYumUpdates(Name):
     LG().Log('DEBUG', "Retrieving update package list using cmd: " + cmd)
     retcode, pkg_list = RunGetOutput(cmd, False, False)
     LG().Log('DEBUG', "Cmd execution time: " + str((time.time() - start_time) / 60))
-    #LG().Log('DEBUG', "Cmd output for update list : " + pkg_list)    
+    LG().Log('DEBUG', "Cmd output for update list : " + pkg_list)    
```

Comment:
make sure pkg_list is of type string.

Reply:
it is string or list will be converted to type str

Reply:
Change Considered